src/jobs/characters.js

```diff
@@ -0,0 +1,68 @@
+const CronJob = require('cron').CronJob;
+import moment from 'moment';
+import models from '../models';
+import { Op } from 'sequelize';
+
+const Character = models.Character;
+
+async function updateCharacter(character) {
+  await Character.update({
+    battlegroup: null,
+    faction: null,
+    class: null,
+    race: null,
+    gender: null,
+    achievementPoints: null,
+    thumbnail: null,
+    spec: null,
+    role: null,
+    talents: null,
+    heartOfAzeroth: null
+  }, {
+    where: {
+      id: character.id
+    }
+  });
+}
+
+async function deleteExpiredCharacters() {
+  Character.findAll({
```

Comment:
Couldnt this just be one Character.update call with the where clause being on blizzardUpdatedAt versus using a .find and then a .update?